src/Kovan-DssSpell.sol

```diff
@@ -66,8 +66,20 @@ contract DssSpellAction is DssAction {
     uint256 constant RAY = 10**27;
     uint256 constant RAD = 10**45;
 
-    address constant MCD_CLIP_UNIV2DAIETH_A      = 0xfcFd4255F67C70Cf5fB534535eBe8152Ba6DC5Cd;
-    address constant MCD_CLIP_CALC_UNIV2DAIETH_A = 0x0Aa53A82182dd60a630A49eCc286b295fEC5Ba98;
+    address constant MCD_CLIP_USDC_A            = 0x09D45087c035DbcD8d6fB5e9d4c5341b9101E626;
+    address constant MCD_CLIP_CALC_USDC_A       = 0xF8D26c26Ac481794E4Aebf4F35B10d8E9748086a;
+    address constant MCD_CLIP_USDC_B            = 0xedFc36f75faafa80e39cd4623def15da6CF2B5C0;
+    address constant MCD_CLIP_CALC_USDC_B       = 0x275076c9c101AF880BD944991258d564FA31D61B;
+    address constant MCD_CLIP_TUSD_A            = 0x9D547d599489B3950485cBa119FC37Bba9c15c13;
+    address constant MCD_CLIP_CALC_TUSD_A       = 0x4AE93701287b8C86f17E5a0Cb4D0732b5ae6EFBD;
+    address constant MCD_CLIP_USDT_A            = 0xBDd2d10dAF8D86dA1f02bB7c7C7841bC9A4F62D4;
+    address constant MCD_CLIP_CALC_USDT_A       = 0xa3a5163Fa4d46D799fE4B036349f0289D69A4445;
+    address constant MCD_CLIP_PAXUSD_A          = 0x3939B686a0A7265512D38Ea3fe700812A703BF31;
+    address constant MCD_CLIP_CALC_PAXUSD_A     = 0x784863edC4C28D73192bf56944D8803c0b5E0CbF;
+    address constant MCD_CLIP_GUSD_A            = 0x448eD0ff4e154C1cBefE2c8057906Dd3dA194dA5;
+    address constant MCD_CLIP_CALC_GUSD_A       = 0x4DD8AaB74a710E7a95937ef1b2618ee76F829Ba6;
+    address constant MCD_CLIP_PSM_USDC_A        = 0xC8Ca47D0AE4193b3f7F813E95669cfB15d922D56;
+    address constant MCD_CLIP_CALC_PSM_USDC_A   = 0x22C3286711bD63D04Da2Ea95C4d7B556B9502a70;
```

Comment:
wards

```haskell
0x09D45087c035DbcD8d6fB5e9d4c5341b9101E626
└─ externally owned account: 0x09D45087c035DbcD8d6fB5e9d4c5341b9101E626


0xF8D26c26Ac481794E4Aebf4F35B10d8E9748086a
└─ externally owned account: 0xF8D26c26Ac481794E4Aebf4F35B10d8E9748086a


0xedFc36f75faafa80e39cd4623def15da6CF2B5C0
└─ externally owned account: 0xedFc36f75faafa80e39cd4623def15da6CF2B5C0


0x275076c9c101AF880BD944991258d564FA31D61B
└─ externally owned account: 0x275076c9c101AF880BD944991258d564FA31D61B


0x9D547d599489B3950485cBa119FC37Bba9c15c13
└─ externally owned account: 0x9D547d599489B3950485cBa119FC37Bba9c15c13


0x4AE93701287b8C86f17E5a0Cb4D0732b5ae6EFBD
└─ externally owned account: 0x4AE93701287b8C86f17E5a0Cb4D0732b5ae6EFBD


0xBDd2d10dAF8D86dA1f02bB7c7C7841bC9A4F62D4
└─ externally owned account: 0xBDd2d10dAF8D86dA1f02bB7c7C7841bC9A4F62D4


0xa3a5163Fa4d46D799fE4B036349f0289D69A4445
└─ externally owned account: 0xa3a5163Fa4d46D799fE4B036349f0289D69A4445


0x3939B686a0A7265512D38Ea3fe700812A703BF31
└─ externally owned account: 0x3939B686a0A7265512D38Ea3fe700812A703BF31


0x784863edC4C28D73192bf56944D8803c0b5E0CbF
└─ externally owned account: 0x784863edC4C28D73192bf56944D8803c0b5E0CbF


0x448eD0ff4e154C1cBefE2c8057906Dd3dA194dA5
└─ externally owned account: 0x448eD0ff4e154C1cBefE2c8057906Dd3dA194dA5


0x4DD8AaB74a710E7a95937ef1b2618ee76F829Ba6
└─ externally owned account: 0x4DD8AaB74a710E7a95937ef1b2618ee76F829Ba6


0xC8Ca47D0AE4193b3f7F813E95669cfB15d922D56
└─ externally owned account: 0xC8Ca47D0AE4193b3f7F813E95669cfB15d922D56


0x22C3286711bD63D04Da2Ea95C4d7B556B9502a70
└─ externally owned account: 0x22C3286711bD63D04Da2Ea95C4d7B556B9502a70

```

Reply:
oh, wait, I did that on mainnet lol

Reply:
:heavy_check_mark: wards

```haskell
0x09D45087c035DbcD8d6fB5e9d4c5341b9101E626
└─ ward: MCD_PAUSE_PROXY


0xF8D26c26Ac481794E4Aebf4F35B10d8E9748086a
└─ ward: MCD_PAUSE_PROXY


0xedFc36f75faafa80e39cd4623def15da6CF2B5C0
└─ ward: MCD_PAUSE_PROXY


0x275076c9c101AF880BD944991258d564FA31D61B
└─ ward: MCD_PAUSE_PROXY


0x9D547d599489B3950485cBa119FC37Bba9c15c13
└─ ward: MCD_PAUSE_PROXY


0x4AE93701287b8C86f17E5a0Cb4D0732b5ae6EFBD
└─ ward: MCD_PAUSE_PROXY


0xBDd2d10dAF8D86dA1f02bB7c7C7841bC9A4F62D4
└─ ward: MCD_PAUSE_PROXY


0xa3a5163Fa4d46D799fE4B036349f0289D69A4445
└─ ward: MCD_PAUSE_PROXY


0x3939B686a0A7265512D38Ea3fe700812A703BF31
└─ ward: MCD_PAUSE_PROXY


0x784863edC4C28D73192bf56944D8803c0b5E0CbF
└─ ward: MCD_PAUSE_PROXY


0x448eD0ff4e154C1cBefE2c8057906Dd3dA194dA5
└─ ward: MCD_PAUSE_PROXY


0x4DD8AaB74a710E7a95937ef1b2618ee76F829Ba6
└─ ward: MCD_PAUSE_PROXY


0xC8Ca47D0AE4193b3f7F813E95669cfB15d922D56
└─ ward: MCD_PAUSE_PROXY


0x22C3286711bD63D04Da2Ea95C4d7B556B9502a70
└─ ward: MCD_PAUSE_PROXY

```

Reply:
Hmmm not sure I'm following this

Reply:
oh ok sorry hadn't refreshed the new comments.